CollAction/Controllers/ProjectsController.cs

```diff
@@ -163,7 +166,7 @@ public ViewResult StartInfo()
                 "Hi!<br>" +
                 "<br>" +
                 "Thanks for submitting a project on www.collaction.org!<br>" +
-                "The CollAction Team will review your project as soon as possible � if it meets all the criteria we�ll publish the project on the website and will let you know, so you can start promoting it! If we have any additional questions or comments, we�ll reach out to you by email.<br>" +
+                "The CollAction Team will review your project as soon as possible � if it meets all the criteria we�ll publish the project on the website and will let you know, so you can start promoting it! If we have any additional questions or comments, we�ll reach out to you by email.<br>" +
```

Comment:
I'm getting invalid unicode characters here in this change? Are you using a mac? We had issues like that before. Undo?